### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by OptimusOpus**
The reverted message has changed

### CODE REVIEW

File: scripts/deploy/contracts/BillBuster.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { deployContract, sleep } from "../utils";
+import { BillBuster } from "../../../build/typechain";
+import { verifyOnEtherscan } from "../../tasks/verifyContractsEtherscan";
+import hre, { ethers } from "hardhat";
+
+export const contractNames = () => ["BillBuster"];
+
+export const constructorArguments = () => [
+  "0xd9BAcC5BccAd9A380001d41Cd234b4D5f33ece76"
```
Comment (RalphLavelle) [inline scripts/deploy/contracts/BillBuster.ts]: Duplicate - can't you use the one from contracts.json?

File: scripts/deploy/contracts/BillBuster.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+];
+
+export const deploy = async (deployer, setAddresses) => {
+  console.log("deploying BillBuster");
```
Comment (RalphLavelle) [inline scripts/deploy/contracts/BillBuster.ts:14]: Want to leave in all these console.logs?

File: scripts/deploy/contracts/BillBuster.ts (hunk 1 above)
Comment (xiangzhe0812) [inline scripts/deploy/contracts/BillBuster.ts]: I think this guy should get from contracts.NETWORK.REMI, so you don't have to change it every time redeploy the contract.

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ import hre, { ethers } from "hardhat";
 export const contractNames = () => ["InvestorVesting"];
 
 const tokenAddress = "0xd9BAcC5BccAd9A380001d41Cd234b4D5f33ece76";
```
Comment (RalphLavelle) [inline scripts/deploy/contracts/InvestorVesting.ts]: See above comment

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 4]
```diff
@@ -49,11 +49,12 @@ export const deploy = async (deployer, setAddresses) => {
     "InvestorVesting",
     constructorArguments(),
     deployer,
-    6
+    4
```
Comment (RalphLavelle) [inline scripts/deploy/contracts/InvestorVesting.ts]: You really shld declare this type of variable so someone else can make sense of it. 
const numberOfFacesInAPyramid = 4. I have no idea what it represents

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 5]
```diff
@@ -6,7 +6,7 @@ import hre, { ethers } from "hardhat";
 export const contractNames = () => ["InvestorVesting"];
 
 const tokenAddress = "0xd9BAcC5BccAd9A380001d41Cd234b4D5f33ece76";
-const startDate = 1660201727;
+const startDate = 1660274000;
```
Comment (xiangzhe0812) [inline scripts/deploy/contracts/InvestorVesting.ts:10]: lets use new Date() + few days, so unit test will always working.

File: scripts/deploy/contracts/InvestorVesting.ts [hunk 6]
```diff
@@ -49,11 +49,12 @@ export const deploy = async (deployer, setAddresses) => {
     "InvestorVesting",
     constructorArguments(),
```
Comment (xiangzhe0812) [inline scripts/deploy/contracts/InvestorVesting.ts:52]: same as this contructorArguments, should get from contracts.NETWORK.REMI

File: scripts/deploy/contracts/InvestorVesting.ts (hunk 5 above)
Comment (OptimusOpus) [reply scripts/deploy/contracts/InvestorVesting.ts:10]: I handle that in the unit test file itself. I think it is nice to have the time hardcoded so you can't accidentally deploy a contact with the incorrect time (as if the starttime is in the past the constructor will throw)

File: scripts/deploy/contracts/BillBuster.ts (hunk 2 above)
Comment (OptimusOpus) [reply scripts/deploy/contracts/BillBuster.ts:14]: Yeah, the deployment process is still quite manual and I find these console.logs help ensure you are running the correct commands

**Review by xiangzhe0812 — APPROVED** [review]

**Review by RalphLavelle — APPROVED** [review]

**Pull request merged**